### ISSUE TITLE
Factoid aliases are, perhaps, too transparent

### ISSUE THREAD

**Issue by dgw** [zigdon/xkcd-Bucket#97]
Say I have an alias: `green eggs => Dr. Seuss`

If a user triggers `green eggs`, Bucket will pick a factoid from `Dr. Seuss` and say it, using `Dr. Seuss` instead of `green eggs`.

For example:

````
<User> green eggs
<Bucket> Dr. Seuss can't possibly be wrong!
````

Ridiculous example aside, the point is that Bucket can seem to be talking about something completely different. Why don't triggered aliases use the name of the alias before the verb, instead of the name of the main factoid group?

**Comment by zigdon** [comment]
It depends what you mean to do when you alias A to B. If the idea is that B
can always be used for A, it makes sense. For anything else, it breaks down.

I feel the example alias is invalid - why would you do that?

**Comment by dgw** [comment]
Yeah, it was a terrible example… my only excuse is that it was really late and I was too tired to come up with anything better.

But the most frequent case I see people "complaining" about on my instance is aliases like `usernick_mobile => usernick`, where triggering a factoid about someone who happens to be on their mobile/alternate nickname spits out a factoid using their primary nick. That case isn't a big deal IMO, but it's the most common.

To be fair, _most_ aliases point to targets that almost exclusively use `<reply>`, so it's hard to dig up a real-life example that isn't nick-related. But I'm sure I'll come across one at some point, and then add it here.
